src/exception/property-exceptions/repo/entity.ts

```diff
@@ -109,7 +110,8 @@ const entityExceptionRepository = {
 		L.debug(`Finding entity exception with program id: ${JSON.stringify(programId)}`);
 		try {
 			// first found document, or null
-			const doc = await EntityExceptionModel.findOne({ programId }).lean(true);
+			const doc = await EntityExceptionModel.findOne({ programId }).lean<EntityException>(true);
+			if (!doc) throw new Error();
```

Comment:
I think will break functionality here: https://github.com/icgc-argo/argo-clinical/blob/05e5caa1944982a8b8ba496fa2fd169272d31427/src/exception/property-exceptions/property-exception-service.ts#L171

Instead of handling the failed case and returning a failed response, this change will throw a database error up the stack to (I think) the default error handlers

Same comment as above, if these changes are being done because they were causing errors, I think this needs it's own ticket / investigation.